cmd.sh

```diff
@@ -38,6 +38,9 @@ clusterCreateCmd=$(printf "%s --min-nodes %s" "$clusterCreateCmd" "$minNodes")
 if [ "$async" = "true" ]; then
   clusterCreateCmd=$(printf "%s --async" "$clusterCreateCmd")
 fi
+if [ "$clusterVersion" != " " ]; then
```

Comment:
why not keep vars outside quotes?
if [ $clusterVersion != " " ]; then

Reply:
might have special characters